skypy/utils/random.py

```diff
@@ -73,3 +74,56 @@ def schechter(alpha, x_min, x_max, resolution=100, size=None, scale=1.):
     lnx_sample = np.interp(u, cdf, lnx)
 
     return np.exp(lnx_sample) * scale
+
+
+def triaxial_axis_ratio(zeta, xi, size=None):
+    r'''axis ratio of a randomly projected triaxial ellipsoid
+
+    Given the two axis ratios `1 >= zeta >= xi` of a randomly oriented triaxial
+    ellipsoid, computes the axis ratio `q` of the projection.
+
+    Parameters
+    ----------
+    zeta : array_like
+        Axis ratio of intermediate and major axis.
+    xi : array_like
+        Axis ratio of minor and major axis.
+    size : tuple of int or None
+        Size of the random draw. If `None` is given, size is inferred from
+        other inputs.
+
```

Comment:
is `Returns` section missing?

Reply:
Yes, let me try and add one
```suggestion

    Returns
    -------
    q : array_like
        Axis ratio of the randomly projected ellipsoid.

```